### DISCUSSION TITLE
The requested resource could not be found

### DISCUSSION THREAD

**Discussion by saini615** [appium/appium#18923] [Q&A] [answered]
I'm facing the error.
selenium.common.exceptions.WebDriverException: Message: The requested resource could not be found, or a request was received using an HTTP method that is not supported by the mapped resource
Please tell me, What causes this error? and what will be the solution?
![Screenshot 2023-07-30 at 3 22 50 PM](https://github.com/appium/appium/assets/49878640/378aaeed-2868-4b26-8d8f-024a4ca3d88d)

**Comment by jlipps** [comment, marked as answer]
Do you see how there's an extra slash in `//session`? Try changing your webdriver Remote URL to not include a trailing slash.

**Reply by Vladik288** [reply]
Как это сделать?

**Comment by CCodeInspect** [comment]
> Do you see how there's an extra slash in `//session`? Try changing your webdriver Remote URL to not include a trailing slash.

赞，我的问题也是类似。
